src/shared/components/tracks/ptmTrackStyles.module.scss

```diff
@@ -0,0 +1,39 @@
+.ptmLegend {
+  ul {
+    list-style: none; /* Remove default bullets */
+
+    li::before {
```

Comment:
These styles are to add custom colors to the bullets in a list.

Reply:
any reason why we couldn't use the regular ul -> li bullets?

Reply:
not a big deal. just curious.  it's ironic.

Reply:
oh, I just copied the logic from here, may not be the best way to do it though: https://www.w3schools.com/howto/howto_css_bullet_color.asp